stetho/src/main/java/com/facebook/stetho/json/ObjectMapper.java

```diff
@@ -273,6 +273,18 @@ private Object getJsonValue(Object value, Class<?> clazz, Field field)
       return convertValue(value, JSONObject.class);
     }
     // hmm we should be able to directly serialize here...
```

Comment:
yes sorry, I meant this comment.  it should be just before the last line's return statement.